shared/default-data/index.ts

```diff
@@ -51,3 +56,47 @@ export function getDefaultData(): ElementConfigType {
     widgetTriggerIcon: ''
   }
 }
+
+export const createDefaultBannerConfig = (
+  presetName: string
+): BannerConfig => ({
+  $version: '1.0.0',
```

Comment:
Let's use this for now.

```suggestion
  $version: '0.0.1',
```